### DISCUSSION TITLE
Oraxen generator support

### DISCUSSION THREAD

**Discussion by YouSeeMeRunning2** [BG-Software-LLC/SuperiorSkyblock2#1100] [Implemented Feature Requests] [answered]
Add the ability to support oraxen custom ores within the cobblestone generator. 

An example would be:
  generator:
    normal:
    - COBBLESTONE:90
    - COAL_ORE:5
    - ORAXEN:RUBY_ORE:5

https://www.spigotmc.org/resources/%E2%80%8D%E2%9C%85-25-%E2%98%84%EF%B8%8F-oraxen-add-items-blocks-armors-hats-food-furnitures-plants-and-gui.72448/ 
https://docs.oraxen.com/developers/api

**Comment by OmerBenGera** [comment, marked as answer]
Implemented, will be uploaded to the official plugin soon.
